src/enums/topics.enum.ts

```diff
@@ -5,5 +5,6 @@ export enum AcademicTopics {
   LanguageDiversity = "Language diversity",
   Git = "Git",
   Next = "Next.js",
-  Solid = "Solid.js"
+  Solid = "Solid.js",
+  Hardware = "Hardware.js",
```

Comment:
do you have any specific topic name rather than `Hardware.js`? its sound is actually cool but I imagine it would be more meaningful if it's like `Hardware - Arduino` or something like that

Reply:
yes, change to Hardware and C++. My mistake.

Reply:
lemme change again to Hardware - C++